src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy

```diff
@@ -451,14 +450,13 @@ class AutoRerouteSpec extends HealthCheckSpecification {
                 .findAll { !untouchableIsls.contains(it) }.unique { [it, it.reversed].sort() }
         islHelper.breakIsls(islsToBreak)
         //move the flow to DOWN status
-        def islToBreak = pathHelper.getInvolvedIsls(flowPath).first()
+        def islToBreak = initialPath.flowPath.getInvolvedIsls().first()
         islHelper.breakIsl(islToBreak)
 
         when: "Generate switchUp event on switch which is not related to the flow"
         def involvedSwitches = pathHelper.getInvolvedSwitches(flowPath)*.dpId
         def switchToManipulate = topology.activeSwitches.find { !(it.dpId in involvedSwitches) }
         def blockData = switchHelper.knockoutSwitch(switchToManipulate, RW)
-        def isSwitchActivated = false
         wait(WAIT_OFFSET) {
             timedLoop(4) {
                 //waiting for the last retry in the scope of flow rerouting due to the ISL failure
```

Comment:
It seems like this code is not refactored: `northbound.getFlowHistory` instead of `flow.retrieveFlowHistory()` and `northbound.getFlowStatus` instead of `flow.retrieveFlowStatus()`, see lines 463-467

Reply:
good catch, thank you